DataFormats/Detectors/TOF/include/DataFormatsTOF/RawDataFormat.save.h

```diff
@@ -0,0 +1,223 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
We want to keep it as a reference for a while since we are trying to align what we have in o2 and in other places.
At some point it will disappear. It is done by purpose

Reply:
But why don't you keep in some private branch?

Reply:
ok. removed